src/Tizen.Diagnostics/Tizen.Diagnostics/DiagnosticsEnumerations.cs

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace Tizen.Diagnostics
+{
+    /// <summary>
+    /// Enumeration for the context types.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public enum ContextType
```

Comment:
What is the point of the enum with only a single value? I see that it is used in `internal DiagnosticsContext(IntPtr handle)`, but there it will assign only ContextType.Crash value.

Reply:
ContextType is going to be extended in the future.